src/Faker/Provider/fr_FR/Company.php

```diff
@@ -166,4 +165,16 @@ protected static function isCatchPhraseValid($catchPhrase)
 
         return true;
     }
+
+    /**
+     * @link http://www.emploi.com/emploi/index-metiers.html
```

Comment:
Too bad. I got the [permission](http://i64.tinypic.com/2eyj49j.png) and they can't guarantee that this list will still be available in the future, as the website will evolve. Even if I remove this list and make another one by hand, I don't think it's allowed to reserve a list of common names.

Anyway, if it's still not OK, I will make one by hand with just few names. Then I'll be sure there wouldn't be any problem.